spec/models/facility_spec.rb

```diff
@@ -2,18 +2,18 @@
 
 RSpec.describe Facility, type: :model do
   describe "Associations" do
-    it { should have_many(:users) }
-    it { should have_many(:blood_pressures).through(:encounters).source(:blood_pressures) }
-    it { should have_many(:blood_sugars).through(:encounters).source(:blood_sugars) }
-    it { should have_many(:prescription_drugs) }
-    it { should have_many(:patients).through(:encounters) }
-    it { should have_many(:appointments) }
-    it { should have_many(:teleconsultations) }
-    it { should have_and_belong_to_many(:teleconsultation_medical_officers) }
-
-    it { should have_many(:registered_patients).class_name("Patient").with_foreign_key("registration_facility_id") }
-    it { should have_many(:assigned_patients).class_name("Patient").with_foreign_key("assigned_facility_id") }
-    it { should have_many(:assigned_hypertension_patients).class_name("Patient").with_foreign_key("assigned_facility_id") }
+    it { is_expected.to have_many(:users) }
```

Comment:
Should we be adopting one over the other going forward?

Reply:
`expect` is the preferred approach, as `should` does a lot of object patching to work. `is_expected` is shorthand for `expect(subject)` 👍